conf/application.conf

```diff
@@ -44,7 +44,7 @@ microservice {
             external-url = "http://localhost:9435"
         }
         agent-client-management-frontend {
-            external-url = "http://localhost:9568/manage-your-tax-agents"
+            external-url = "/agent-client-relationships/manage-your-tax-agents"
         }
```

Comment:
Can be removed along with corresponding val in AppConfig

Reply:
good point!